.github/workflows/ci_integration_tests_ubuntu.yaml

```diff
@@ -93,7 +93,7 @@ jobs:
     - uses: actions/checkout@v3
     - name: build ckb and run bats cli test
       run: |
-        make build
+        make prod
```

Comment:
I'm not sure, why change `build` to `prod`, it's not related to the issue we are resolving in this PR?

Reply:
`make build` and `make prod` are distinct. The former does not include features such as `deadlock_detection` and `with_sentry`. And CKB packages are released using `make prod`. Therefore, I believe it would be more appropriate to use `make prod` for the `make cli-test` command.

Reply:
sound reasonable, just a little bit weird to mixed in this PR 😆